tests/data/demo/v1/fornebu/shop_scenario.yaml

```
@@ -11,17 +11,38 @@
   commands: "[name]default"
   source: resync
   attribute_mappings_override: []
+  output_definition:
+    - "[name]market price"
+    - "[name]market sale"
+    - "[name]plant production"
+    - "[name]plant consumption"
+    - "[name]reservoir water value"
+    - "[name]reservoir energy conversion factor"
```

I think we only need these:
```suggestion
    - "[name]market price"
    - "[name]plant production"
    - "[name]plant consumption"
```
Should (of course) be updated for the other scenarios too.
Seems a bit annoying to have to define the full list on every scenario - can we add this list as a default value in data_model_configuration.yaml? I guess we don't want to change the data model to have a separate type for lists of output definitions, but it seems like that would have been easier to use.